bin/check_design.py

```diff
@@ -2,7 +2,7 @@
 
 #######################################################################
 #######################################################################
```

Comment:
I think I'll get this going for Eager soon too 👍 Looks fab :-)

Reply:
Thanks. It does work really well, restricts user error and gives you experiment-level info :smiley: